config/tsconfig.base.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "strict": true,
+    "noImplicitAny": true,
+    // "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "strictBindCallApply": true,
+    "strictPropertyInitialization": true,
+    // "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    // "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    // "isolatedModules": true,
+    "noEmit": true, // ensure that nobody can accidentally use this config for a build
+    "sourceMap": true,
+    "removeComments": true,
+    "downlevelIteration": true,
+    // "paths": {
+    //   "@project-leek/*": ["./packages/*/src"] // TODO check if helpful
+    // }
+  }
+}
```

Comment:
why do you keep the commeted sections?

Reply:
Most of these lines should be enabled in the future, but are throwing a lot of errors currently. I will have an other look on them after the new setup established itself a bit. Also check [this comment](#discussion_r554424734).